### PR TITLE
added icons to dashboard/homepage and create file for friend group an…

### PR DESCRIPTION
**Pull request by lnovitz**
…d add friend

### CODE REVIEW

File: frontend/src/views/Dashboard.js [hunk 1]
```diff
@@ -26,13 +28,29 @@ function Dashboard() {
...
+      <span class="material-symbols-outlined">
+        account_circle
+      </span>
         <div className={styles.title}>Welcome</div>
+        <p>to</p>
+        <div className={styles.title}>Phone a Friend!</div>
+        <div>
+          <span class="material-symbols-outlined" >
+            group
+          </span>
```
Comment (lnovitz) [inline frontend/src/views/Dashboard.js:40]: Looks like you may need to add `Link` and we also need to setup the routes somewhere (per this page in stackoverflow https://stackoverflow.com/questions/44387318/linking-button-to-another-page) I don't know how to do that so I'll let @ValentinaBacherer take the lead on setting up routes 😬

**Comment by lnovitz** [comment]
Screenshot 2023-10-17 at 8 19 24 AM
Loving the icons! I just pushed 1 commit on your branch to fix a syntax compile error, so make sure to pull from the remote before continuing work @heejung-hong

File: frontend/src/views/Dashboard.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ import styles from '../styles/Dashboard.module.css';
 import { PassageAuthGuard } from '@passageidentity/passage-react';
 import { usePassageUserInfo } from '../hooks/';
 import LogoutButton from '../components/LogoutButton';
+import { BrowserRouter, Routes, Route, Outlet, Link } from 'react-router-dom';
+import FriendGroup from './FriendGroup';
```
Comment (lnovitz) [inline frontend/src/views/Dashboard.js:6]: I don't know if we're supposed to be using the `Dashboard.js` file or if we're supposed to create a new view. The `Dashboard.js` is part of the Passage authentication I think... idk.

Comment (heejung-hong) [reply frontend/src/views/Dashboard.js:6]: The Dashboard.js originally indicated that it was the homepage so I was trying to modify it to look like our homepage.  After the routing and link functionality is established, I can work on the css styling.

File: frontend/src/views/Dashboard.js (hunk 1 above)
Comment (heejung-hong) [reply frontend/src/views/Dashboard.js:40]: Yes, I did try adding Link and set up routes in different ways.  The issue I'm having is when I wrap the icon in the Link, the icon disappears from view.  I'm going to try to download the icon image and add it to the code and try linking the image for routing.

**Pull request closed without merging**